New version: GXBeam v0.5.2

JuliaRegistrator opened the pull request:
- Registering package: GXBeam
- Repository: https://github.com/byuflowlab/GXBeam.jl
- Created by: @Cardoza2
- Version: v0.5.2
- Commit: d9543e9130f1fd9a5be82c8ccc0b9d58c4610097
- Reviewed by: @Cardoza2
- Reference: https://github.com/byuflowlab/GXBeam.jl/issues/127#issuecomment-2813799775
- Description: Pure Julia Implementation of Geometrically Exact Beam Theory
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598e08684ee7e716aad3b3f1a4fc7fe5d9f814fbd727772acd8d26b430679dc7b914b8fcc4d273ad1f6791e4fd2f42c4205ddcd5a0dca182e6d2db7a5bced5f65a768de9fd2962f0c585f4e36d3c7c9c3ca7723c2393f9caac201dba32786b19609bb0e8ff8c71f74d68c28402898caaf16b2e958bc1d77096c73ffb8eb5a0830e7 -->

Cardoza2 commented:
I'm not sure I understand the automerge guidelines decision/explanation. It's not happy about FillArrays and ForwardDiff. But I think I have upper-bounded compat requirements for both of those packages... and since those packages have upper bounds, wouldn't that imply that they have a finite number of breaking changes? Is it perhaps the other packages that just specify a single version number? 

Thanks for the help!

Here's my current compat: 

```
[compat]
julia = "1.6"
ArnoldiMethod = "0.0.4 - 0.5"
ChainRulesCore = "1"
FLOWMath = "0.3 - 0.5"
FillArrays = "0.13 - 2"
ForwardDiff = "0.10 - 1"
ImplicitAD = "0.3.1 - 0.4"
IterativeSolvers = "0.9 - 0.10"
LinearMaps = "2 - 4"
NLsolve = "4"
OffsetArrays = "1"
ReverseDiff = "1"
Roots = "1 - 3"
SciMLBase = "1"
SparseArrays = "1.6 - 2"
SparseDiffTools = "2"
StaticArrays = "1.9 - 2"
UnPack = "1"
WriteVTK = "1.21 - 2"
```

Cardoza2 commented:
I may have figured it out on my end. My tests are running. I'll update the registration request when they're done. [noblock]

JuliaRegistrator committed:
New version: GXBeam v0.5.2

UUID: 974624c9-1acb-4ad6-a627-8ac40fc27a3e
Repo: https://github.com/byuflowlab/GXBeam.jl.git
Tree: 09d8f79cc213dd50492f83d73911d1ec3aeb47c5

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
